src/storage/gpu_device_storage.h

```diff
@@ -62,7 +62,7 @@ inline void* GPUDeviceStorage::Alloc(size_t size) {
 #endif  // MXNET_USE_NCCL
   cudaError_t e = cudaMalloc(&ret, size);
   if (e != cudaSuccess && e != cudaErrorCudartUnloading)
-    throw std::bad_alloc();
+    LOG(FATAL) << cudaGetLastError();
```

Comment:
Does cudaGetLastError return a string or an error code?

Reply:
Good catch! I have changed it to call cudaGetErrorString